### DISCUSSION TITLE
Log in and out of multiple Fief accounts

### DISCUSSION THREAD

**Discussion by davidbrochart** [fief-dev/fief#233] [Q&A]
With several accounts created on Fief, when loging out of one and loging in on the other, I get and error saying that "jupyverse.fief.dev redirected you too many times". Clearing the cookies seems to help, but I was wondering if this could be improved?

**Comment by frankie567** [comment]
Hmm, that shouldn't happen. Are you sure you're going through `https://jupyverse.fief.dev/logout?redirect_uri=XXX` when logging out your user?

It's important to make sure the `fief_session` cookie is cleared. If properly logged out, the only cookies that may remain are `fief_login_hint` and `fief_csrftoken`, which shouldn't cause any issue.

**Reply by davidbrochart** [reply]
I'm talking about users in Fief itself, e.g. at https://fief.fief.dev/admin/auth/login.

**Reply by frankie567** [reply]
Oh! Right indeed, I can reproduce the issue.

That's clearly a bug. Something weird happens with the `fief_admin_session` which makes Fief ping-pong redirect with the previous and new workspace 🤔
